apollo-router/src/spec/schema.rs

```diff
@@ -23,7 +23,7 @@ use crate::*;
 /// A GraphQL schema.
 #[derive(Debug, Default, Clone)]
 pub(crate) struct Schema {
-    string: Arc<String>,
+    pub(crate) string: Arc<String>,
```

Comment:
There must be a better name.

Reply:
I'll rename to `raw_sdl`

Reply:
[a414cc3](https://github.com/apollographql/router/pull/2173/commits/a414cc313a528ec40d3660b60705fd916b34c0eb)